lib/busbar_cli/commands/resize.rb

```diff
@@ -4,7 +4,26 @@ module Resize
 
     included do
       desc 'resize APP ENV [COMPONENT_TYPE] NODE_TYPE',
-           'Change the current node type of an environment or a component'
+           'Change the current node type of an environment or a component (Available types: 1x, 2x, 4x, 1x.standard, 2x.standard, 4x.standard, 8x.standard, 1x.performance, 2x.performance, 4x.performance, 8x.performance)'
+      long_desc <<-LONGDESC
+           Change the current node type of an environment or a component.
+
+           Available types: 1x, 2x, 4x, 1x.standard, 2x.standard, 4x.standard, 8x.standard, 1x.performance, 2x.performance, 4x.performance, 8x.performance
```

Comment:
If I were to read this description in the terminal, would it automatically wrap this long line in the shell or cut it off with ellipses (`…`)? If it's the latter, then I strongly recommend making this line no longer than 80 chars.